codice/ddf
# DDF-4201 Upgrade spring to address vulnerabilites

#### What does this PR do?

Upgrades spring to address CVE-2018-1275 and
CVE-2018-1270, and removes suppressions.

#### Who is reviewing it? 

@emmberk @peterhuffer 

#### Select relevant component teams: 

@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.

@clockard
@tbatie 
@rzwiefel

#### How should this be tested?

Build and install DDF. Keep an eye on the logs and make sure that all the necessary bundles come up. Make sure that all the blacklisted features aren't installed and confirm that spring-core version 5.0.4.RELEASE isn't included in the distribution. spring-core-ldap has also been upgraded so verify that ldap login still works. (message me if you need help with this)

#### What are the relevant tickets?

[DDF-4201](https://codice.atlassian.net/browse/DDF-4201)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
